src/static/css/homepage.css

```diff
@@ -0,0 +1,26 @@
+.asd {
+    padding: 60px;
+    text-align: center;
+    background: #6C7A89;
```

Comment:
pull a colour from https://github.com/the-template-project/root/issues/2

Reply:
done -- went with #343a40 (onyx)